### PR TITLE
Booleans should be true and false

### PR DESCRIPTION
**Pull request by TheDoubleK**
This allows booleans to save and display values.

It also allows checkboxes to be pre-checked for values that exist in the values passed in.

### CODE REVIEW

**Comment by quis** [comment]
Going to need to update the docs since this changes the way selection-button is used. Can help you with this tomorrow.

**Comment by TheDoubleK** [comment]
Docs updated.  I checked an inline radio but left the regular radios unchecked, and ticked a couple of checkboxes.

File: pages_builder/pages/forms/selection-buttons.yml [hunk 1]
```diff
@@ -25,16 +25,17 @@ examples:
...
       -
         label: Public
-        checked: true
       -
         label: Private
   -
     type: checkbox
     name: categories
     question: Choose a category
+    value: ['Accounting and finance','Collaboration']
```
Comment (quis) [inline pages_builder/pages/forms/selection-buttons.yml]: Sorry for nitpicking—and I know this works—but it's more YAML-onic to do:
```YAML
value:
  - Accounting and finance
  - Collaboration
```

Comment (TheDoubleK) [inline pages_builder/pages/forms/selection-buttons.yml]: No worries, yep it doesn't look quite right now you say it.

**Pull request merged**